### ISSUE TITLE
Loss problem, don't have mrcnn_bbox_loss and mrcnn_mask_loss.

### ISSUE THREAD

**Issue by hardworking-jws** [matterport/Mask_RCNN#2603]
100/100 [==============================] - 136s 1s/step - loss: 2.0010 - rpn_class_loss: 0.4009 - val_loss: 1.7868 - val_rpn_class_loss: 0.3553
Epoch 2/100
100/100 [==============================] - 117s 1s/step - loss: 1.1283 - rpn_class_loss: 0.2530 - val_loss: 1.5249 - val_rpn_class_loss: 0.2836
Epoch 3/100
100/100 [==============================] - 116s 1s/step - loss: 0.8029 - rpn_class_loss: 0.2133 - val_loss: 1.3147 - val_rpn_class_loss: 0.2652
Epoch 4/100
100/100 [==============================] - 117s 1s/step - loss: 0.6996 - rpn_class_loss: 0.2019 - val_loss: 1.5031 - val_rpn_class_loss: 0.2805

My problem is there was no mrcnn_bbox_losss、mrcnn_mask_loss and rpn_bbox_loss show in my terminal. And the same at tensorboard in browser.
Please help me. Thanks!

**Comment by alexander-pv** [comment]
Hi, I faced several issues with MaskRCNN including losses issues and decided to update the model to TensorFlow v2. You can check the current version of this fork [here](https://github.com/alexander-pv/maskrcnn_tf2).
